cfe_internal/enterprise/federation/federation.cf

```diff
@@ -410,6 +410,27 @@ body contain contain_transport_user
     useshell => "true";
 }
 
+bundle monitor measure_psql
+{
+  measurements:
+    "/var/log/postgresql.log"
+      handle => "psql_lock_wait_before_acquisition",
+      stream_type => "file",
+      data_type => "real",
+      history_type => "weekly",
+      units => "ms",
+      match_value => acquired_Lock,
+      comment => "Are comments significant?";
```

Comment:
Comments are not significant with respect to how mission portal graphs stuff. That was just junk from my testing. Maybe change the comment or yank it?

Reply:
yank, will do.